Dekarbonizacia priemyslu

LenkaSedlackova opened the pull request:
- [x] zamyslieť sa nad vložením infa z [akčného plánu NVS do 2026](https://rokovania.gov.sk/RVL/Material/28497/1), ktorý bol schválený 13.6.23 a samotnej [NVS](https://www.mhsr.sk/uploads/files/YBN0ndkU.pdf?csrt=2310365313461346377)
- [x] skontrolovať prosím moje výpočty v [^zachyt] a [^vodik]
- [x] fix autorstva (bo na tom padá build)

LenkaSedlackova committed:
založenie a preklad po 90

LenkaSedlackova committed:
upload piktogramov/animácií

LenkaSedlackova committed:
upload ilustračného obr

LenkaSedlackova committed:
po 129

LenkaSedlackova committed:
návrat sekcie Neutralita + prehádzanie obsahu + vloženie dekarbonizácie priemyslu

LenkaSedlackova committed:
po 197

LenkaSedlackova committed:
po 226 (a výpočet pre SR v CCS poznámke)

výpočet urobený podľa ČR predlohy (CRF2A1 pre SK krát spodná/horná hranica pre spotrebu elektriny a cca priemer uvedený)

LenkaSedlackova committed:
finiš prekladu a výpočet v referencii [vodik]

pomocou infografiky /elektrina-sr (4571+589 GWh voda+solár)

LenkaSedlackova committed:
update autorstva a drobné úpravy textu

LenkaSedlackova committed:
slug fix

LenkaSedlackova committed:
jazyk brush

vrq-github commented:
Pridala som info zo stratégie, výpočty ja kontrolovať nechcem to je dúfam n a Jenu

LenkaSedlackova committed:
zmena názvu, slugu apod na "resers" a update linku

LenkaSedlackova committed:
slug fix

LenkaSedlackova committed:
slug fix 2.0

LenkaSedlackova committed:
oprava linku a slug fix 2.0

LenkaSedlackova committed:
pridanie SK reálií

LenkaSedlackova committed:
zapracovanie Adamových komentárov

jankrcal committed:
Oprava strukturovaných autorů

jankrcal committed:
auto-format

jankrcal commented:
Zatím jsem opravil autorství, výpočty zkotroluji později...

jankrcal committed:
Aktualizace web-core (po merge)

jankrcal committed:
Oprava odkazu na sankey grafiku elektřina

jankrcal committed:
Úprava výpočtu pro CCS u cementu

Počítal jsem procenta z čisté spotřeby (protože je to zde relevantnější
metrika) a zaokrouhlil 1,9 na 2.

jankrcal committed:
Odstranění roku u výpočtu zelené elektřiny

To je náchylné na chyby při aktualizaci grafiky. Výroba se nevyvíjí tak
rychle, aby bylo uvedení roku nutné.

jankrcal committed:
CSS pro layout ilustrací

jankrcal committed:
Drobná oprava

jankrcal reviewed (approved):
Opravil jsem technické věci a drobné obsahové věci. Lenko, prosím zkontroluj jazykově moje drobné zásahy.

Pokud jste udělaly celkovou jazykovou kontrolu (tedy četl to někdo jiný, než kdo to překládal), můžete dělat merge.

jankrcal committed:
Aktualizace data explaineru

LenkaSedlackova committed:
oprava mena a SK preklad

LenkaSedlackova commented:
@jankrcal [toto](https://github.com/faktaoklimatu/web-sk/pull/111/commits/1492e865170c9a6ba7efd0233b866b0f9cbfbe62) sa prosím bude zobrazovať na stránke, t.j. to má byť po slovensky alebo to je iba nejaká interná technická záležitosť?
text prešiel Adam ako jeho autor, tak keď vyriešime faktí tím, môžeme mergovať

jankrcal commented:
Mně šlo o kontrolu slovenštiny. Nechci Adama podceňovat, ale preferoval bych rodilého mluvčího :)

Ty názvy skupin lidí se na stránce zobrazovat nebudou, ale klidně je pro pořádek můžeš přeložit do slovenštiny.

LenkaSedlackova commented:
> Mně šlo o kontrolu slovenštiny. Nechci Adama podceňovat, ale preferoval bych rodilého mluvčího :)

@marcel-otruba prosím spravíš nám jazykovú review? ja som možno niečo prehliadla po toľkých čítaniach

jankrcal commented:
@marcel-otruba je zaměstnán jinýma úkolama, tak na něj prosím nedávej jazykové review. Pokud jsi to psala ty, může na to prosím kouknout Viera? Příp. pokud to psala Viera, tak už mi stačí, že jsi to toilkrát četla sama :)

LenkaSedlackova commented:
oká, tak budeme veriť mojim očiam a mergujem :) ešte Andy je Slovenka, inak na tieto jazykové reviews bohužiaľ nemáme ľudí...
